recipes/osmanip/all/CMakeLists.txt

```diff
@@ -6,23 +6,28 @@ conan_basic_setup(KEEP_RPATHS)
 
 find_package(arsenalgear REQUIRED CONFIG)
 
-set(osmanip_src
```

Comment:
It would be simpler to use [target_sources](https://cmake.org/cmake/help/latest/command/target_sources.html) here instead of introducing a variable for the purpose.

Reply:
Done. Thanks!